src/TrezorKeyring.ts

```diff
@@ -1,17 +1,32 @@
-const { EventEmitter } = require('events');
-const ethUtil = require('@ethereumjs/util');
-const HDKey = require('hdkey');
-const TrezorConnect = require('@trezor/connect-web').default;
-const { TransactionFactory } = require('@ethereumjs/tx');
-const { transformTypedData } = require('@trezor/connect-plugin-ethereum');
+import { EventEmitter } from 'events';
```

Comment:
Nit: Our filename convention is to use `kebab-case.ts`, rather than `CamelCase.ts`. That's what we've been doing for new files at least; we haven't converted the core monorepo to kebab case yet since this was established.